java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -514,7 +515,7 @@ public int getID() {
       }
     }
 
-    private final long nativeHandle;
+    final long nativeHandle;
```

Comment:
>final long nativeHandle; [](http://example.com/codeflow?start=4&length=24)

I suggest introducing getters.

Reply:
I don't understand the C# reference? I can make package private get methods in Java, but it didn't seem worth the effort.

Reply:
Yes, this is exactly what I have in mind. From the C# reference, we had to re-work a few times where the native handles are stored or how they are obtained. Getters would allow doing it without refactoring. Making private handles public is not good.

Reply:
Done. The package private fields are now private, and there are package private accessors.